src/parser.py

```diff
@@ -63,9 +63,9 @@ def create_output_file():
             
 def main():
     print("Processing problem statements...")
-    process_statements("data/statements.txt")
```

Comment:
Why is this directory being changed?

Reply:
Oh sorry my vs code environment defaults the working directory to the root folder instead of src, so on my end I have to specify src in the file path. I'll re commit with src removed

Reply:
I think it committed the fix. Let me know if it didn't work